README.md

```diff
@@ -224,6 +224,39 @@ If your repo has a different command to run for migrations, you can configure it
 `DeclareSchema.db_migrate_command = "bundle exec rails db:migrate_immediate"`
 ```
 
+## The `belongs_to` Relation and Indexes
+By default, `declare_schema` creates an index for `belongs_to` relations. If an index
```

Comment:
Rather than "relations" I'd go with "associations". (I think "relation" is relational DB theory is actually the table, not the foreign keys across table.)

Reply:
Right you are, that's loose terminology on my part.